client/features/friends/controllers/friendsController.js

```diff
@@ -0,0 +1,86 @@
+angular.module('friends', ['friends.services', 'userId.services'])
+
+.controller('friendsController', function ($scope, $location, $rootScope, friendsFactory) {
+  /*
+   * hasCoveys can have a value of 'true', 'false', or 'error'
+   * the view will automatically change based on the value.
+   */
+  $scope.showFriends = 'true';
+  $scope.friends = [];
+  // Routes user to the specified covey (based on coveyId)
+  $scope.removeFriend = (friendId) => {
+    let friendIndex = $scope.friends.length;
```

Comment:
Instead of using friendIndex as both the index of the removed friend and the length of the array, can we keep them separate and use a 'break' inside the for loop? This should satisfy the same purpose and make the code more clear.